bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -449,6 +449,10 @@ <h2 class="primary go">{{ _('Help Mozilla users') }}</h2>
   <button id="scroll-prompt">{{ _('Discover more') }}</button>
 </main>
 
+{% if waffle.switch('fundraiser-home-takeover') and l10n_has_tag('eoy_fundraising_2015') %}
```

Comment:
Ok `eoy_fundraising_takeover` ? Also, please note that there is different copy in the copy and design bug than that on the design provided in https://bugzilla.mozilla.org/show_bug.cgi?id=1214038#c15

Also, not sure, is the copy final? https://bugzilla.mozilla.org/show_bug.cgi?id=1214034#c3 @m-alexis